src/queryAnalysis/utils.go

```diff
@@ -23,11 +23,12 @@ func LoadQueries(arguments args.ArgumentList) ([]models.QueryDetailsDto, error)
 	for i := range queries {
 		switch queries[i].Type {
 		case "slowQueries":
-			queries[i].Query = fmt.Sprintf(queries[i].Query, arguments.FetchInterval, arguments.QueryCountThreshold, arguments.QueryResponseTimeThreshold)
+			queries[i].Query = fmt.Sprintf(queries[i].Query, arguments.FetchInterval, arguments.QueryCountThreshold,
+				arguments.QueryResponseTimeThreshold, config.TextTruncateLimit)
 		case "waitAnalysis":
-			queries[i].Query = fmt.Sprintf(queries[i].Query, arguments.FetchInterval, arguments.FetchInterval, arguments.QueryCountThreshold)
+			queries[i].Query = fmt.Sprintf(queries[i].Query, arguments.FetchInterval, arguments.QueryCountThreshold, config.TextTruncateLimit)
```

Comment:
arguments.FetchInterval updated to arguments.QueryCountThreshold ?

Reply:
no. 
Since I converted all the %d and %s into variable substitution and added the declaration of variables on top of each query we just need to substitute the variable(FetchInterval) value once instead of multiple times(that we were doing earlier)